admin_guide/configuring_authentication.adoc

```diff
@@ -524,32 +531,12 @@ is configured to trust the `X-Remote-User` header.
   SSLProxyCACertificateFile /etc/pki/CA/certs/ca.crt
   SSLProxyMachineCertificateFile /etc/pki/tls/certs/authproxy.pem
```

Comment:
is it possible to limit the ssl client cert config to just the ProxyMatch stanza? I just want to avoid it getting picked up by other ProxyPass paths